includes/admin/settings/class-wc-settings-shipping.php

```diff
@@ -64,81 +64,105 @@ public function get_sections() {
 			}
 		}
 
-		return apply_filters( 'woocommerce_get_sections_' . $this->id, $sections );
+		return $sections;
 	}
 
 	/**
-	 * Get settings array.
+	 * Is WC_INSTALLING constant defined?
+	 * This method exists to ease unit testing.
+	 *
+	 * @return bool True is the WC_INSTALLING constant is defined.
+	 */
+	protected function wc_is_installing() {
```

Comment:
Technically this isn't actually necessary. The Jetpack Constants package we're using lets you change constants for unit tests already. This may not have been the case when you wrote the PR I think.